exercises/part3-iterators/index.js

```diff
@@ -16,7 +16,15 @@ Instructions: Write a function which counts the number of times a value occurs i
 Example: countItem(['a', 'b', 'a'], 'a') should return 2
 ===================== */
 
-let countItem = (arr, item) => {};
+let countItem = (arr, item) => {
+  let results = 0;
+  for (let i = 0; i < arr.length; i++) {
+      if (arr[i] == item) {
+          result += 1;
```

Comment:
You call it `result` here, but define it as `results`. This small typo is causing the whole program to fail.
```suggestion
          results += 1;
```